controllers/host/platformnetworks.go

```diff
@@ -838,9 +870,110 @@ func (r *HostReconciler) ReconcilePlatformNetworkBootstrap(
 	addrpool_instance *starlingxv1.AddressPool,
 	system_info *cloudManager.SystemInfo) error {
 
-	err := r.ReconcileNetworkAndAddressPools(client, network_instance, addrpool_instance, system_info)
-	if err != nil {
-		return err
+	return r.ReconcileNetworkAndAddressPools(client, network_instance, addrpool_instance, system_info)
+
+}
+
+func (r *HostReconciler) TriggerHostLockStrategy(host_instance *starlingxv1.Host, host_id string, host_personality string) error {
```

Comment:
Suggest to rename it as "LockHostRequest"

Reply:
Done.